ArduCopter/ReleaseNotes.txt

```diff
@@ -1,5 +1,83 @@
 ArduPilot Copter Release Notes:
 ------------------------------------------------------------------
+Copter 4.1.0-beta1 14-Apr-2021
+Changes from 4.0.7
+1) EKF changes:
+    a) EKF3 is default estimator (EKF2 is available as an option)
+    b) External AHRS/IMU support (e.g. VectorNav)
+    c) Gaussian Sum Filter (GSF) allows emergency yaw correction using GPS
+    d) GPS-for-yaw (dual F9 UBlox GPS can provide yaw)
+    e) Lane switching logic improvements
+    f) Sensor affinity (improves sensor failure redundancy)
+    g) Source switching for GPS/Non-GPS transitions (see EK3_SRCx_ parameters)
+    h) Yaw estimation and reliability improvements
+2) Control and Navigation improvements:
+    a) SCurves for waypoint navigation
+    b) Auto mdoe arming and takeoff (see AUTO_OPTIONS)
+3) 6DoF frame support
+4) Object avoidance:
+    a) BendyRuler hesitancy improvements
+    b) Intel Realsense 435/455 camera support (companion computer required)
+    c) Simple avoidance backs away from obstacles (see AVOID_BACKUP_SPD parameter)
+    d) Simple avoidance accel limited (see AVOID_ACCEL_MAX parameter)
+    e) Simultaneous Dijkstra and BendyRuler path planning
+    f) Obstacle database now 3D
+    g) Obstacle filtering improvements
+    h) Vertical BendyRuler
```

Comment:
- Minimum altitude before Simple Avoidance is switched on (see AVOID_ALT_MIN parameter)

- Filter Obstacle near the ground using downward facing range finder (see PRX_IGN_GND parameter)

- Ignore obstacles near home in OA Database (see OA_DB_ALT_MIN)

- Proximity library supports 3D data (Through OBSTACLE_DISTANCE_3D mavlink message)

- Avoidance consumes 3D data (not sure how the above two should be worded, or if they should be even included. But it's a significant change!)

- Various Bug fixes (Like we added the face timeout, which was an issue in 4.0, and various issues were also fixed in Simple Avoidance)